components/v2/JoinGame/JoinGameList/JoinGameList.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+import { JoinGameListPropsI } from './JoinGameListProps';
+import styles from './JoinGameList.module.scss';
+import { getRulesContract } from 'gameApi';
+import { JoinGameCard } from '../JoinGameCard';
+import { gameEntitiesQuery } from 'queries';
+import { useQuery } from '@apollo/client';
+const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
```

Comment:
Is there a file with global constants, like `ZERO_ADDRESS` ? In case there isn't one, it's time to start it.

Reply:
done